pointerevents/pointerevent_gotpointercapture_before_first_pointerevent-manual.html

```diff
@@ -57,10 +57,16 @@
                         }
                         else {
                             if (pointerdown_event.pointerId === event.pointerId) {
```

Comment:
We don't expect multiple pointers in this test, right? In that case, this condition should be replaced with:
<code>
if(receivedPointerDown) {
  test id equality; ...
}
</code>